roadmap/implementers-guide/src/types/network.md

```diff
@@ -8,7 +8,12 @@ These types are those that are actually sent over the network to subsystems.
 type RequestId = u64;
 type ProtocolVersion = u32;
 struct PeerId(...); // opaque, unique identifier of a peer.
-struct View(Vec<Hash>); // Up to `N` (5?) chain heads.
+struct View {
+	// Up to `N` (5?) chain heads.
+	heads: Vec<Hash>,
```

Comment:
It feels like we should encode the size restriction into the typesystem somehow, just as self-protection. Possibly like

```rust
struct LimitedSizeVec<T> {
  items: SmallVec<T>,
  capacity: usize,
}
```

That's probably out of scope of this PR, though.

Reply:
yes, out of scope